lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -395,6 +398,49 @@
       - Specifies the order of operations when rolling out an updated task.
       - Corresponds to the C(--update-order) option of C(docker service create).
       - Requires API version >= 1.29.
+  rollback_delay:
+    type: int
+    default: 0
```

Comment:
Adding this here, and to the other examples should fix CI
```suggestion
    default: 0
    version_added: "2.8"
```

Reply:
Also, you should reorder this to first have `description`, then `type`, then `default`, then `choices`, then `version_added`. I've changed all options in all docker_* modules accordingly (in #52422), which unfortunately requires you to rebase (together with some other PRs which got merged today).